src/main/java/org/mskcc/cmo/messaging/model/LinkedPatient.java

```diff
@@ -0,0 +1,53 @@
+package org.mskcc.cmo.messaging.model;
+
+import java.io.Serializable;
+import org.neo4j.ogm.annotation.GeneratedValue;
+import org.neo4j.ogm.annotation.Id;
+import org.neo4j.ogm.annotation.NodeEntity;
+import org.neo4j.ogm.annotation.Relationship;
+
+/**
+ *
+ * @author ochoaa
+ */
+@NodeEntity
+public class LinkedPatient implements Serializable {
+    @Id @GeneratedValue
+    private Long id;
+    private String linkedPatientName;
+    private String linkedSystemName;
+    @Relationship(type="PX_TO_PX", direction=Relationship.OUTGOING)
+    private PatientMetadata patientMetadata;
+
+    public LinkedPatient() {}
+
+    public LinkedPatient(String linkedPatientName, String linkedSystemName) {
+        this.linkedPatientName = linkedPatientName;
+        this.linkedSystemName = linkedSystemName;
+    }
+
+    public String getLinkedPatientName() {
```

Comment:
Is name what we now call value in the example graph on dashi-dev - the actually uuid or alias id?

![image](https://user-images.githubusercontent.com/366003/99286380-c4b45700-2806-11eb-9e24-1f7630399b7d.png)

Reply:
In this graph the `LinkedPatient` nodes are the outermost nodes linked to a `PatientMetadata` node.